pkg/config/configsections/crd.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2021 Red Hat, Inc.
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+package configsections
+
+// Crd defines a CustomResourceDefinition config filter.
+type Crd struct {
```

Comment:
does this really need to be a structure? even if we add other ways of matching CRDs, it would be a separate list, right? e.g. 
```
targetCrds:
  groups:
    - a
    - b
  labels:
    - name: lname
      value: lvalue
```